src/iac_scan_runner/project_config.py

```diff
@@ -0,0 +1,142 @@
+import pymongo
+import bson.json_util as json_util
+from bson.json_util import dumps
+import json
+from datetime import datetime
+import os
+import uuid
+
+
+class ProjectConfig:
+    def __init__(self):
+        """
+        Initialize new user config result database, collection and client
+        """        
+        self.connect_db()
+                
+                 
+    def connect_db(self):
+        """
+        Initialize new project collection connection into scan result database
+        """
+                        
+        try:
+            connection_string = os.environ['MONGODB_CONNECTION_STRING']  
+            
+            if connection_string:
+                self.myclient = pymongo.MongoClient(connection_string)
+                
+                # TODO: Extract these names in a separate object or variable
+                self.mydb = self.myclient["scandb"]
+                self.mycol = self.mydb["configs"]
+                self.connection_problem = False
+        
+        # TODO: Consider more specific exceptions     
+        except Exception as e:
+            print("Configuration persistence not available, error: {e}")
+            self.myclient = None
+            self.mydb = None
+            self.mycol = None
+            self.connection_problem = True
+
+    def parse_json(self, data):
+        try:
+            return json.loads(json_util.dumps(data))
+        except Exception as e:
+            print("Could not parse JSON, error: {e}")
```

Comment:
This won't print exception, but this will:

```suggestion
            print(f"Could not parse JSON, error: {e}")
```

Reply:
Fixed